helper/chunkify.js

```diff
@@ -0,0 +1,30 @@
+function chunkify(message, maxChars){
+    let lines = message.split(/\n/);
+    let chunks = [];
+    let numChars = 0;
+    for (let i = 0; i < lines.length; i++){
+        numChars = lines[i].length;
+        if (numChars < maxChars){
+            chunks.push(lines[i]);
+        } else {
+            let loc = lines[i].lastIndexOf(" ");
+            let subchunk = "";

```

Comment:
Might be better to stick to one format of naming, so either `numChars` & `subChunk` or `num_chars` and `sub_chunk`, better to stick to a variable naming format

Reply:
Resolved in latest fix